projects/lib/src/configurator/palette/palette.component.html

```diff
@@ -57,6 +57,6 @@ <h6 class="d-inline" i18n>Drag & drop an existing component</h6>
 </ng-template>
 
 <!-- Modal -->
-<uib-modal [title]="modal?.title" [show]="!!modal" (close)="onModalClose($event)">
-  <div *ngTemplateOutlet="modal?.configurator.templateRef; context: { $implicit: modal }"></div>
+<uib-modal *ngIf="modal" [title]="modal.title" [show]="!!modal" (close)="onModalClose($event)">
```

Comment:
Is the `*ngIf` necessary? It looks like the component is meant to be activated via the `[show]` input.

Reply:
Same comment as above